templates/pioneer-citp-news-disinfo/measurements/measurements.1.schema.json

```diff
@@ -63,9 +75,20 @@
                     "domain": { "type": "string" },
                     "classification": {
                         "type": "string",
-                        "description": "deprecated, use pageClassification"
+                        "description": "deprecated, use classifierResults"
+                    },
+                    "pageClassification": {
+                        "type": "integer",
+                        "description": "deprecated, use classifierResults"
+                    },
+                    "classifierResults": {
+                        "type": "object",
+                        "properties": {},
```

Comment:
ditto

Reply:
cool, I did that and it looks like this now --
```
                    "classifierResults": {
                      "additionalProperties": {
                        "type": "integer"
                      },
                      "description": "additional properties are the name of the classifier, with the classification result as the value",
                      "type": "object"
                    },
```
seems reasonable to me, is this what you would expect?

Reply:
Oops, ok, I got rid of that line and the new generated schema looks like this,
```
                    "classifierResults": {
                      "additionalProperties": {
                        "type": "integer"
                      },
                      "description": "additional properties are the name of the classifier, with the classification result as the value",
                      "type": "object"
                    },
```
seem right to you?

Reply:
An oops on my part 😅. That looks right to me, once you push it and I retrigger integration, the bigquery schema that's posted should have a key/value struct instead of a string.